decisions/0015-env-triad-terms.md

```diff
@@ -1,52 +1,124 @@
+  with the exception of host-associated samples, including plant-associated samples.
+- `env_local_scale` will consist of subclasses
+  of [material entity [BFO:0000040]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FBFO_0000040?lang=en),
+  minus biome [ENVO:00000428]
+  and [environmental material [ENVO:00010483]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00010483)
+    - It's expected that additional branches of the is_a hierarchy will subtracted,
+      like [chemical entity [CHEBI:24431]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCHEBI_24431)
+    - The possibility of subtracting leaf nodes is under consideration. Since we are programmatically building candidate
+      tables with multiple forms of evidence, the smaller query result offered by subtraction may not be of much added
+      benefit
+    - Subtraction over relationships other than is_a, subtraction of classes based on substrings in their labels, or
+      other targets subtraction are possible, but may also be of limited benefit
+    - @turbomam
+      has [prototyped these workflows in Makefiles](https://github.com/microbiomedata/external-metadata-awareness/blob/main/Makefiles/soil-env_local_scale.Makefile).
+      @sierra-moxon will explore options for making the workflows more accessible to the whole group, with a single
+      application that takes a single configuration file, or with (a) Jupyter notebook(s)
+- `env_medium` will consist of subclasses environmental material [ENVO:00010483]
 
 For **soil** environment (MIxS Extension)
-- `env_broad_scale` will exclude aquatic biome
-  - Included terms will be separated into leaf nodes & non-leaf nodes & evaluated manually and additional query changes will be made as needed.
-  - Expectation is that lead nodes will be removable.
+
+- `env_broad_scale` will
+  exclude [aquatic biome [ENVO:00002030]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00002030)
+    - See comments about the cost and benefits of excluding leaf nodes above
 - `env_local_scale` ????? TBC
-- `env_medium` will be textually filtered for classes with soil or soils in their name or description.
-  - Will evaluate if adding stemming is needed.
+- `env_medium`
+    - Subclass hierarchy-based exclusion _could_ also be used here, for
+      example [hydrocarbon-based environmental material [ENVO:2000045]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_2000045?lang=en)
+    - Textual filtering over labels, with or without regular expressions or stemming, _could_ also be used
+        - food, water and ice terms could be removed without stemming
+        - stemming with the pattern 'ferment' could be used to remove terms with fermented or fermenting in their labels
```

Comment:
Ok, but did you? If not take it out. I asked you to add details about what you **did**

Reply:
not could do.